Lib/test/test_sqlite3/test_cli.py

```diff
@@ -1,14 +1,22 @@
 """sqlite3 CLI tests."""
 import sqlite3
+import sys
+import textwrap
 import unittest
+import unittest.mock
```

Comment:
This is unused, no?

Reply:
It's used on line 203.
If you run `python -m test test_sqlite3`, tests would pass because something else sets the `mock` attribute on `unittest`. But running only `python -m test test_sqlite3.test_cli` failed.